SYCL/Reduction/reduction_nd_N_vars.cpp

```diff
@@ -125,10 +130,22 @@ int runTest(T1 IdentityVal1, T1 InitVal1, BinaryOperation1 BOp1,
      auto In3 = InBuf3.template get_access<access::mode::read>(CGH);
      auto In4 = InBuf4.template get_access<access::mode::read>(CGH);
 
+#ifdef TEST_SYCL2020_REDUCTIONS
+     auto Redu1 = sycl::reduction(OutBuf1, CGH, IdentityVal1, BOp1);
+     auto Redu2 = sycl::reduction(OutBuf2, CGH, IdentityVal2, BOp2);
+     auto Redu3 = sycl::reduction(OutBuf3, CGH, IdentityVal3, BOp3);
+     auto Redu4 = sycl::reduction(Out4, IdentityVal4, BOp4);
+#else
      auto Out1 = OutBuf1.template get_access<Mode1>(CGH);
      auto Out2 = OutBuf2.template get_access<Mode2>(CGH);
      accessor<T3, 0, Mode3, access::target::global_buffer> Out3(OutBuf3, CGH);
 
+     auto Redu1 = ONEAPI::reduction(Out1, IdentityVal1, BOp1);
```

Comment:
Are these reductions deprecated in SYCL 2020 mode? It is not clear why they are exclusive (one or other but not both in the same app).
Duplicating compile/run lines will cause duplication for execution time which I would like to avoid.

Reply:
ONEAPI::reduction is not mentioned in SYCL-2020 and is not deprecated, ONEAPI::reduction exists in parallel with sycl::reduction now

Reply:
both sycl::ONEAPI::reduction and sycl::reduction may exist in the same function.

Reply:
I understand what you mean now. Having both sycl::reduction and ONEAPI::reduction used without macros would reduce compilation time.
I can do additional changes to address that.

Reply:
Fixed in e6df87c69e290cd2ad11b5466a98ed8ed8507d16